pallets/crowdloan-claim/src/weights.rs

```diff
@@ -48,33 +48,41 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(12 as Weight))
 			.saturating_add(T::DbWeight::get().writes(5 as Weight))
 	}
+
```

Comment:
It might seem outrageous, and maybe we open an issue for this, but it would be great to comment the weight functions to get a sense what we are measuring here (like what is `sr25519`?)